packages/dds/ordered-collection/src/consensusOrderedCollection.ts

```diff
@@ -140,6 +141,7 @@ export class ConsensusOrderedCollection<T = any>
 		await this.submit<IConsensusOrderedCollectionAddOperation<T>>({
 			opName: "add",
 			value: valueSer,
```

Comment:
Open an ADO task to stop writing this in a release or two?

Reply:
Probably falls under https://dev.azure.com/fluidframework/internal/_workitems/edit/7149/